pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.echobox</groupId>
   <artifactId>ebx-linkedin-sdk</artifactId>
-  <version>1.0.3</version>
+  <version>1.0.4</version>
```

Comment:
This should not be set to 1.0.4 until after the release otherwise we'll release a 1.0.4 version.

Reply:
thank you . I have fixed this